int32 index
---
string name
geometry_msgs/Pose pose